ipl-auction-react/src/control/outLinedTextField.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import PropTypes from "prop-types";
+import "./outLinedTextField.css";
+
+const OutLinedTextFiled = (props) => {
+  const {
+    id,
+    type,
+    value,
+    checked,
+    disabled,
+    placeholder,
```

Comment:
can we manage placeholder in outLinedTextField itself?

Reply:
Yes, we can manage placeholder in outLinedTextField itself.